lib/events/team_join.js

```diff
@@ -1,4 +1,27 @@
-module.exports = (data, cb) => {
+//const Group = require('../models/group')
+const slack = require('slack')
+const bluebird = require('bluebird')
+const chat = bluebird.promisifyAll(slack.chat)
+
+const cocTxt = process.env.COC_URL ? `<${process.env.COC_URL}|Code of Conduct>` : 'Code of Conduct'
+const welcome_message_text = process.env.WELCOME_MESSAGE_TEXT || `Welcome! Review the ${cocTxt} before participating in the community.`
+const send_welcome_message = process.env.SEND_WELCOME_MESSAGE === 'FALSE' ? false : true
```

Comment:
We don't have a way to do that right now, which kinda sucks but was in my mental to-do list for Soon™.

You did the bulk of the work for this feature too so what I meant is that we could merge this with default messages, and then customize it once the feature is up. I'm still not sure what the best way to do that is.